gulpfile.js

```diff
@@ -269,40 +272,46 @@ function resolveInstallerExePath(artifactType) {
   return path.join(zaRoot, artifactName + '-' + pjson.version + artifactType + '-installer.exe');
 }
 
-// download all the installer dependencies so we can package them up into the .exe
-gulp.task('prefetch', function(cb) {
-  let counter=0;
+// prefetch all the installer dependencies so we can package them up into the .exe
+gulp.task('prefetch', function() {
+  let promises = new Set();
+  if (!fs.existsSync(prefetchFolder)) {
+     mkdirp(prefetchFolder);

```

Comment:
mkdirp is an async function, it doesn't cause probles at this point but should be moved to separate gulp task similarly to _create-dist-win-dir_